### PR TITLE
[CMD-535] dataengineering 2.2.0

### PR DESCRIPTION
**Pull request by better-sachin**

### CODE REVIEW

File: base/Dockerfile.dataengineering [hunk 1]
```diff
@@ -24,20 +24,6 @@ RUN adduser --home /home/${USER} \
...
-  /tmp/scripts/common/
-RUN :                                                         \
-  && apt-get update                                           \
-  # Run common install scripts                                \
-  && apt-get install libopenblas-dev wget git less bash curl openssl coreutils ca-certificates jq tar -y \
-  && chmod +x /tmp/scripts/common/install-rds-certificates.sh \
-  && chmod +x /tmp/scripts/common/install-sops.sh             \
-  && /tmp/scripts/common/install-rds-certificates.sh          \
-  && /tmp/scripts/common/install-sops.sh                      \
-  && ln -s /usr/include/locale.h /usr/include/xlocale.h
```
Comment (better-sachin) [inline base/Dockerfile.dataengineering:39]: not sure what locale.h and xlocal.h is used for

Comment (tokuchan) [reply base/Dockerfile.dataengineering:39]: locale.h is a [library](https://www.tutorialspoint.com/c_standard_library/locale_h.htm) that is used to make it easier to set things like date formats, currency, and numerical formats up for a specified region. Be sure that none of your libraries (such as numpy) need it, before you remove it.

**Review by tokuchan — APPROVED** [review]
LGTM, but make sure you don't actually need the locale libraries.

File: base/Dockerfile.dataengineering (hunk 1 above)
Comment (better-sachin) [reply base/Dockerfile.dataengineering:39]: added the ls back

**Pull request merged**